ui/components/MesheryPerformanceComponent.js

```diff
@@ -115,18 +119,21 @@ class MesheryPerformanceComponent extends React.Component {
     if (name === 'url' && event.target.value !== '') {
       this.setState({ urlError: false });
     }
-    if (name === 't' && (event.target.value.toLowerCase().endsWith('h')
```

Comment:
Why this check is not needed 🤔 ?

Reply:
because when you change this drop-down will execute other function handleDurationChange